RubeesEat/Model/DB/DbPersonRepository.cs

```diff
@@ -0,0 +1,92 @@
+using System.Data;
+
+namespace RubeesEat.Model.DB;
+
+public class DbPersonRepository(IDbConnectionFactory connectionFactory) : IPersonRepository
+{
+    public IReadOnlyList<Person> GetAll()
+    {
+        var persons = new List<Person>();
+
+        using var connection = connectionFactory.CreateDbConnection();
+        using var command = connection.CreateCommand();
+
+        command.CommandText = "SELECT PersonID, FirstName, LastName FROM Persons ORDER BY LastName ASC;";
+        command.CommandType = CommandType.Text;
+
+        connection.Open();
+
+        using var reader = command.ExecuteReader();
+
+        while (reader.Read())
+        {
+            persons.Add(new Person
+            (
+                reader.GetGuid(0),
+                reader.GetString(1),
+                reader.GetString(2)
+            ));
+        }
+
+        return persons.AsReadOnly();
+    }
+
+    public void Add(Person person)
+    {
+        using var connection = connectionFactory.CreateDbConnection();
+        using var command = connection.CreateCommand();
+
+        command.CommandText = "INSERT INTO Persons (PersonID, FirstName, LastName) VALUES (@PersonID, @FirstName, @LastName);";
+        command.CommandType = CommandType.Text;
+
+        var personIdParam = command.CreateParameter();
+        personIdParam.ParameterName = "@PersonID";
+        personIdParam.Value = person.Id;
+        command.Parameters.Add(personIdParam);
```

Comment:
We repeat the two operations of creating + setup a command and creating a parameter quite often. To remove code duplication please introduce extension methods to simplify the usage to `CreateCommand("SELECT ...")` and `CreateParameter("@PersonID", person.Id)`